CHANGES.rst

```diff
@@ -72,6 +72,10 @@ astropy.io.registry
 astropy.io.votable
 ^^^^^^^^^^^^^^^^^^
 
+- Corrected a spurious warning issued for the ``value`` attribute of the
+  ``<OPTION>`` element in VOTable, as well as a test that erroneously
+  treated the warning as acceptable.  [#9470]
```

Comment:
This needs to go in the "bug fix" sub-section. But please hold off on moving it until we can be sure of the milestone. Thanks!

Reply:
No problem.  Thanks for being patient.

Reply:
@pllim I don't know whether @tomdonaldson will have time, but I posted some pre-digested snippets from the VOTable standard in a comment last night to make that review easier.

Reply:
Thank you, that'll certainly help!